docs/internal-pipeline/rules_results_output.md

```diff
@@ -0,0 +1,272 @@
+* `reports`: list of rules that detect any problem on given cluster
+* `skips`: list of rules that have been skipped because not all required information was available on checked cluster
+* `pass`: list of rules that passes all conditions (i.e. rules without any problem/issue detected)
+* `info`: list of rules that return info messages only 
+* `fingerprints`: ?
+* `analysis_metadata`: information about Insights Rules settings, context, and analysis duration 
+
+### Format of `system` attribute in `Report` node
+
+TBD (not used in internal data pipeline)
+
+### Format of `reports` attribute in `Report` node
+
+This attribute contains list of rules that detect any problem on given cluster.
+Each element in the list is represented as a node with seven attributes:
+
+* `rule_id`: rule name and a key
+* `component`: fully-qualified name of the rule (unique)
+* `type`: information that issue or issues have been detected by this rule
+* `key`: a key that selects the variant of issue (one rule can detect more different issues)
+* `tags`: tags assigned to the rule
+* `links`: links to documentation, Knowledge Base article etc.
+
+An example:
+
+```json
+        "reports": [
+            {
+                "rule_id": "tutorial_rule|TUTORIAL_ERROR",
+                "component": "ccx_rules_ocp.external.tutorial_rule.report",
+                "type": "rule",
+                "key": "TUTORIAL_ERROR",
+                "details": {
+                    "type": "rule",
+                    "error_key": "TUTORIAL_ERROR"
+                },
+                "tags": [],
+                "links": {}
+            }
+        ]
+```
+
+### Format of `skips` attribute in `Report` node
+
+This attribute contains list of rules that have been skipped because not all
+required information was available on checked cluster. Each element in the list
+is represented as a node with four attributes:
+
+* `rule_fqdn`: fully-qualified name of the rule (unique)
+* `reason`: reason why the rule was skipped
+* `details`: detailed information about the rule and the condition to skip it
+* `type`: information that this rule was skipped
+
+An example:
+
+```json
+        "skips": [
+            {
+                "rule_fqdn": "ccx_rules_ocp.ocs.check_ocs_version.report",
+                "reason": "MISSING_REQUIREMENTS",
+                "details": "All: ['ccx_ocp_core.specs.must_gather_ocs.OperatorsOcsMGOCS'] Any: ",
+                "type": "skip"
+            },
+            {
+                "rule_fqdn": "ccx_rules_ocp.ocs.check_pods_scc.report",
+                "reason": "MISSING_REQUIREMENTS",
+                "details": "All: ['ccx_ocp_core.specs.must_gather_ocs.PodsMGOCS'] Any: ",
+                "type": "skip"
+            }
+        ]
+```
+
+### Format of `pass` attribute in `Report` node
+
+This attribute contains list of rules that passes all conditions (i.e. rules without any problem/issue detected)
+
+### Format of `info` attribute in `Report` node
+
+TBD
```

Comment:
Are these TBDs on purpose, or did you maybe forget to update this part of the docs?

Reply:
It was a copy/paste from previous file, tbh